addons/web/static/src/legacy/js/chrome/abstract_action.js

```diff
@@ -145,7 +159,12 @@ const AbstractAction = Widget.extend(ActionMixin, {
             if ('title' in this.controlPanelProps) {
                 this._setTitle(this.controlPanelProps.title);
             }
+//            if ('view_description' in this.controlPanelProps) {
+//                this._setViewDescription(this.controlPanelProps
+//                .view_description);
+//            }
             this.controlPanelProps.title = this.getTitle();
+//            this.controlPanelProps.view_description = this.getViewDescription();
```

Comment:
```suggestion
            this.controlPanelProps.title = this.getTitle();
```
Sorry, I missed this one in my initial review